modules/flags/FeatureFlags.js

```diff
@@ -16,20 +16,19 @@ class FeatureFlags {
      * @param {boolean} flags.receiveMultipleVideoStreams - Signal support for receiving multiple video streams.
      */
     init(flags) {
-        this._runInLiteMode = Boolean(flags.runInLiteMode);
+        this._runInLiteMode = Boolean(flags?.runInLiteMode);
 
-        this._sourceNameSignaling = Boolean(flags.sourceNameSignaling);
-        this._receiveMultipleVideoStreams = Boolean(flags.receiveMultipleVideoStreams);
-        this._sendMultipleVideoStreams = Boolean(flags.sendMultipleVideoStreams);
-        this._ssrcRewriting = Boolean(flags.ssrcRewritingOnBridgeSupported);
+        this._sourceNameSignaling = flags?.sourceNameSignaling ?? true;
+        this._receiveMultipleVideoStreams = flags?.receiveMultipleVideoStreams ?? true;
+        this._sendMultipleVideoStreams = flags?.sendMultipleVideoStreams ?? true;
+        this._ssrcRewriting = flags?.ssrcRewritingOnBridgeSupported;
```

Comment:
I'd leave `Boolean(...)` for this one.

Reply:
for _ssrcRewriting ? That flag has changed and was hoping James would fix it and define the default behavior in his PR .